CHANGELOG.md

```diff
@@ -26,7 +26,7 @@ This changelog keeps track of work items that have been completed and are ready
 
 ### Improvements
 
-- **General**: TODO ([#TODO](https://github.com/kedacore/http-add-on/issues/TODO))
+- **Operator**: Provide support to allow HTTP scaler to work alongside other core KEDA scalers ([#489](https://github.com/kedacore/http-add-on/issues/489))
```

Comment:
@JorTurFer i'd say this is a "general" feature and not an improvement, but up to you

Reply:
Yeah, let's move it to `New` section and rename it to **General**